src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -112,7 +112,9 @@ public MappingProfiles()
                 .ForMember(dest => dest.Releases, m => m.Ignore())
                 .ForMember(dest => dest.Topic, m => m.Ignore());
             CreateMap<Release, Data.Model.Release>()
-                .ForMember(dest => dest.Publication, m => m.Ignore());
+                .ForMember(dest => dest.Publication, m => m.Ignore())
+                .ForMember(dest => dest.TimeIdentifier, m => m.MapFrom(r => r.TimePeriodCoverage))
+                .ForMember(dest => dest.Year, m => m.MapFrom(r => int.Parse(r.ReleaseName)));
```

Comment:
It's strange that it doesn't already map this, perhaps because there's no setter on the Content side.
What's stopping you `m.MapFrom(r => r.Year)` here?

Reply:
You're right - Year is already being mapped - there was a unit test for that it is passing. I think it's just TimeIdentifier that isn't being set. So I'll just remove the Year map line completely.